src/pages/locationPage.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Card from '../components/Card'
+import location from '../data.json'
+import Header from '../components/Header'
+
+LocationPage.propTypes = {
+  title: PropTypes.string,
+}
+
+export default function LocationPage({ title }) {
+  return (
+    <Grid>
+      <Header>
+        <h2>Best Locations</h2>
+      </Header>
+      <div>
+        {location.map(location => {
+          const { id, attributes } = location
+          return (
+            <Card
+              key={id}
+              image={attributes.main_image_path}
+              title={attributes.title}
+              ratingimage={attributes.rating_image_path}
+              text={attributes.text}
+            />
+          )
+        })}
```

Comment:
Could be shortened to this:
```suggestion
        {location.map(({id, {main_image_path, rating_image_path, title, text}}) => (
            <Card
              key={id}
              image={main_image_path}
              title={title}
              ratingimage={rating_image_path}
              text={text}
            />
          )
}
```